<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSci</class>
 <widget class="QDialog" name="AddSci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>218</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="line_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_yob">
        <property name="text">
         <string>Year of birth</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="line_yob"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="toggle_alive">
        <property name="text">
         <string>Alive</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="toggle_dead">
        <property name="text">
         <string>Dead</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_yod">
        <property name="text">
         <string>Year of death</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="line_yod"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_gender">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_gender">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toggle_alive</sender>
   <signal>toggled(bool)</signal>
   <receiver>toggle_dead</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggle_alive</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_yod</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggle_dead</sender>
   <signal>toggled(bool)</signal>
   <receiver>toggle_alive</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
